tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature

```diff
@@ -36,13 +36,13 @@ Feature: move folders
     Given user "user1" has logged in using the webUI
     And the user has browsed to the files page
     When the user batch moves these folders into folder "simple-empty-folder" using the webUI
-      | name                 |
-      | simple-folder        |
+      | name                  |
+      | simple-folder         |
```

Comment:
it seems like these two lines have been changed accidentally, :thinking:

Reply:
fixed